### ISSUE TITLE
[Documentation] erreur dans les exemples de getWeaponEffects()

### ISSUE THREAD

**Issue by yaelMagnier** [leek-wars/leek-wars-client#492]
> if (targets & EFFECT_TARGET_NOT_CASTER) debug('N'affecte jamais le lanceur');

devrais être

> if (targets & EFFECT_TARGET_NOT_CASTER) debug('N\'affecte jamais le lanceur');

**Comment by PerthuisQuentin** [comment]
Je vois aucune différence entre les deux :)

**Comment by yaelMagnier** [comment]
@PerthuisQuentin Oui, j'avais oublié que github échapais les backslash. C'est modifié
